### ISSUE TITLE
The plan®

### ISSUE THREAD

**Issue by yajo** [OCA/docs#5]
- [x] Move this repo to OCA.
- [x] Set up Netlify in OCA side.
- [x] Check Decap CMS works after moving to new site. Tested in https://github.com/OCA/docs/pull/6.
- [ ] Do some more realistic examples, including a full collaboration flow between several contributors, reviewers, ...
- [ ] Include some metadata (versions, maintainer, license, repo, weblate link, runboat link, maturity). Possibly fetch it by JS using Github API. Otherwise, the bot fills it statically. https://github.com/OCA/docs/issues/10
- [ ] Update readme template so that modules readmes link for docs and hide the rest of the important readme parts under a "Details" button. This could be enabled with an option of oca-gen-addons-readme.
- [ ] Put our logo and brand styles.
- [ ] Make it beautiful. 💄 
- [x] #7
- [ ] Import docs from pre-existing modules. (Maybe done by the bot automatically?)
- [ ] Update maintainer-tools in oca repos template.
- [ ] Add a ✏️ button on each page, for easy editing.
- [ ] Set up an official domain such as docs.odoo-community.org.
  - [ ] Reconfigure Netlify.
  - [ ] Check Decap CMS works with the new domain.
  - [ ] Transfer OAuth Github app ownership from @yajo to @oca.
- [ ] Document very well how to document very well.
- [x] #9
- [ ] Announce this.
- [ ] Test the new flow for a couple of months.
- [ ] If it is satisfactory, remove https://odoo-community.org/shop/

**Comment by florenciafrigieri2** [comment]
@yajo , Jairo, with Julie we propose to add the following steps to the plan, regarding the /shop replacement:

- [ ] Add the version link to Github to the documentation page of the module (in order to see in which versions the module is available and, if needed, go to the module in Github and download it).
- [ ] Verify if we keep the information showed on the /shop like: Licence, Maintainer, Repository.
- [ ] Evaluate what we do with the buttons for: Weblate, Runboat and Maturity (we keep them in the module's readme in Github or we move them to the docs repo?). Maybe this decision depends on how we solve showing the available versions of the module. 

Please, do not hesitate to add more information (or modify something according to your ideas).
cc @sbidoul @TumbaoJu

**Comment by yajo** [comment]
OK, I added all in a single step:

>  Include some metadata (versions, maintainer, license, repo, weblate link, runboat link, maturity). Possibly fetch it by JS using Github API. Otherwise, the bot fills it statically.

**Comment by sbidoul** [comment]
I think the "update the bot" task is too high in the list and this should be done as one of the last things, after, for instance, "display addons metadata" or "update the readme generator", and even maybe "make it beautiful".

The reason is that I would like to gain confidence this docs site will become a good replacement for the apps store before this is used at scale, and avoid the risk of having the two tools to maintain forever.

**Comment by sbidoul** [comment]
I have reordered that tasks a bit according to my previous comment.

**Comment by dreispt** [comment]
I suggest adding a "MVP" top item to the list, with sub items on what is pending to make it possible.
One of those would be to document instruction on how to use this.

**Comment by yajo** [comment]
I think what you ask is already done, @dreispt.

The MVP is live at https://oca-docs.netlify.app/ and ready to accept contributions, and the point of documenting how to use it is already listed as "Document very well how to document very well."

We have a bare-bones contribution guide at https://oca-docs.netlify.app/contributing/ FWIW.

**Comment by ivs-cetmix** [comment]
Hi team, thank you for the really great job!

I have a question regarding this:
> * [ ]  If it is satisfactory, remove https://odoo-community.org/shop/

Will there be a "Download" button on the module page so visitors could download a module instantly?

**Comment by yajo** [comment]
> Will there be a "Download" button on the module page so visitors could download a module instantly?

I don't think so. Probably it will be more like "Download: `pip install odoo-addon-partner-fax`".

**Comment by ivs-cetmix** [comment]
> > Will there be a "Download" button on the module page so visitors could download a module instantly?
> 
> I don't think so. Probably it will be more like "Download: `pip install odoo-addon-partner-fax`".

I see. Well, I think a button is a crucial part so we should definitely find a way to make it easy to download a module for non-technical people.

**Comment by yajo** [comment]
That would involve logic, and this is a static repo, so most chances is that won't happen. Anyway, I'm not too sure about how often is that feature really used. I guess most people just install through git or pip. Still you can download the zip from github, uncompress and copy the module folder, in case you really need it that way.

**Comment by ivs-cetmix** [comment]
> That would involve logic, and this is a static repo, so most chances is that won't happen. Anyway, I'm not too sure about how often is that feature really used. I guess most people just install through git or pip. Still you can download the zip from github, uncompress and copy the module folder, in case you really need it that way.
> 
> In any case, it's a very latest step for the plan. We can see details later on that.

I would say that from what I can see now minimum 70% of people will try to download it. We should keep in mind that this static site is mostly for those non technical people. Or technical but non-yet-OCA-aware ones. Because those OCA-profs are using pip and know most of the OCA repos already.

I think possible solution can be generating some static link  to the "download" controller served by some special OCA module that will do the job.

**Comment by zeroheure** [comment]
> minimum 70% of people will try to download it. We should keep in mind that this static site is mostly for those non technical people.

I don't think that downloading a module will help non-technical people, because they won't be able to install it in Odoo

**Comment by ivs-cetmix** [comment]
> > minimum 70% of people will try to download it. We should keep in mind that this static site is mostly for those non technical people.
> 
> I don't think that downloading a module will help non-technical people, because they won't be able to install it in Odoo

The idea is that besides those non-technical ones there are also technical that are not familiar with the OCA flow. We should not make their life complicated. Sometimes a simple "Download" link does the job)

**Comment by TumbaoJu** [comment]
This project has been abandoned due to community objections on the fact of having part of the doc elsewhere and not with the code and of having only one version of the doc per version.

We will work on developing a tool that will help edit or write the Read me for non-technical people instead.
